appeals/web/src/server/appeals/personal-list/personal-list.mapper.js

```diff
@@ -352,24 +352,51 @@ export function mapAppealStatusToActionRequiredHtml(appeal, isCaseOfficer = fals
 			return [lpaStatementAction, ipCommentsAction].filter(Boolean).join('<br>');
 		}
 		case APPEAL_CASE_STATUS.FINAL_COMMENTS: {
-			const lpaReceived = lpaFinalCommentsStatus === 'received';
-			const appellantReceived = appellantFinalCommentsStatus === 'received';
+			const finalCommentsDueDate = appealTimetable?.finalCommentsDueDate;
+			const isFinalCommentsDueDatePassed = finalCommentsDueDate
+				? dateIsInThePast(dateISOStringToDayMonthYearHourMinute(finalCommentsDueDate))
+				: false;
 
-			if (!lpaReceived && !appellantReceived) {
-				return 'Awaiting final comments';
-			}
+			if (isFinalCommentsDueDatePassed) {
+				const hasValidFinalCommentsAppellant =
+					documentationSummary.appellantFinalComments?.representationStatus &&
+					documentationSummary.appellantFinalComments?.representationStatus ===
+						APPEAL_REPRESENTATION_STATUS.VALID
+						? true
+						: false;
+				const hasValidFinalCommentsLPA =
+					documentationSummary.lpaFinalComments?.representationStatus &&
+					documentationSummary.lpaFinalComments?.representationStatus ===
+						APPEAL_REPRESENTATION_STATUS.VALID
+						? true
+						: false;
```

Comment:
No need for the ternary here as the `===` operator will return a boolean, and also since if `representationStatus` is undefined it won't be equal to `APPEAL_REPRESENTATION_STATUS.VALID`, you can remove that part of the check.

```suggestion
				const hasValidFinalCommentsAppellant = documentationSummary.appellantFinalComments?.representationStatus === APPEAL_REPRESENTATION_STATUS.VALID;
				const hasValidFinalCommentsLPA = documentationSummary.lpaFinalComments?.representationStatus === APPEAL_REPRESENTATION_STATUS.VALID;
```

Reply:
done